lib/matplotlib/_mathtext_data.py

```diff
@@ -1066,7 +1068,7 @@
     'hermitmatrix'             : 8889,
     'barvee'                   : 8893,
     'measuredrightangle'       : 8894,
-    'varlrtriangle'            : 8895,
+    'lrtriangle'               : 8895,
```

Comment:
```suggestion
    'varlrtriangle'            : 8895,
```

Reply:
I'll add lrtriangle then. DIdn't make sense to have varlrtriangle if we don't have lrtriangle...

Reply:
I'm wondering if one should script the source of the STIX documentation and add all those symbols? The problem is just to add them to the documentation...

Reply:
Reverted to varlrtriangle. Any additional symbols will be in another PR.